src/showcase/app/app-routing.module.ts

```diff
@@ -20,15 +20,25 @@ const routes: Routes = [
           sessionStorage.setItem(modeSessionStorageKey, offBrandColorMode);
         }
 
+        let variantPrimaryColor;
         if (sessionStorage.getItem(modeSessionStorageKey) === offBrandColorMode) {
+          variantPrimaryColor = '#06348b';
```

Comment:
Do variables not work here? Or could you at least parse the variable by getComputedStyle?

Reply:
`getComputedStyle(document.documentElement).getPropertyValue('--sbb-color-red')` could be used after the if/else.

Reply:
Thanks for the hint. I fixed it